README.md

```diff
@@ -1,3 +1,25 @@
-# S-BLH-roman-numeral-converter
-2nd project of java script
+<<<<<<< HEAD
+# Roman-Numeral-Converter 🚀
 
+## Project Description 📝
+- 2nd project of js
+- its a converter , that can change latin numbers to roman numbers
+
+## Technologies Used 🛠️
+- HTML
+- CSS
+- JavaScript
+
+## Usage 🎯
+to use this application, you only have to write the greek numbers and then click on the convert, to change them to roman numbers
+
+## Demo 📸
+![Demo]:![alt text](<Screenshot 2024-06-30 144151-1.png>)
```

Comment:
- I can not see your screenshot in your project folder and also you used here wrong use it like this:
- First, add your screenshot to your project folder
- Then just copy and paste this and add the name of your screenshot:
```
![Image](HERE_REMOVE THE TEXT AND ADD YOUR OWN IMAGE NAME AND FORMAT EXAMPLE screenshot.png)
```